<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WMain</class>
 <widget class="QMainWindow" name="WMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>WMain</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="HIC::ApplicationGLWidget" name="glWidget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>621</width>
      <height>411</height>
     </rect>
    </property>
   </widget>
   <widget class="HIC::RobotControl" name="robotControl" native="true">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>0</y>
      <width>151</width>
      <height>541</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="rotXSlider">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>445</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="rotYSlider">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>446</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="rotZSlider">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>447</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="rotXLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>420</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Rotate X axis</string>
    </property>
   </widget>
   <widget class="QLabel" name="rotYLabel">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>420</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Rotate Y axis</string>
    </property>
   </widget>
   <widget class="QLabel" name="rotZLabel">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>420</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Rotate Z axis</string>
    </property>
   </widget>
   <widget class="QLabel" name="transZLabel">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>480</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Translate Z axis</string>
    </property>
   </widget>
   <widget class="QLabel" name="transXLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>480</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Translate X axis</string>
    </property>
   </widget>
   <widget class="QLabel" name="transYLabel">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>480</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Translate Y axis</string>
    </property>
   </widget>
   <widget class="QSlider" name="transYSlider">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>509</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-99</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="transXSlider">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>508</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-99</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="transZSlider">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>510</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-99</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="text">
    <string>new</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>load</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>save</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>HIC::ApplicationGLWidget</class>
   <extends>QWidget</extends>
   <header>applicationglwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>HIC::RobotControl</class>
   <extends>QWidget</extends>
   <header>robotcontrol.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
